samples/Hystrix.Dotnet.Samples.AspNet/Hystrix.Dotnet.Samples.AspNet/Global.asax.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Http;
+using System.Web.Routing;
+
+namespace Hystrix.Dotnet.Samples.AspNet
+{
+    public class WebApiApplication : System.Web.HttpApplication
+    {
+        protected void Application_Start()
+        {
+            GlobalConfiguration.Configure(WebApiConfig.Register);
```

Comment:
In the old classic .net application we also needed the following code. Is it no longer required?

```
// ignore route for hystrix.stream httphandler
RouteTable.Routes.Ignore("hystrix.stream/{*pathInfo}");
```